adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -125,6 +127,7 @@ var AdyenCheckoutHybris = (function () {
                 if (!this.afterPay.state.isValid) {
                     window.alert("Please fill all the details");
                     this.afterPay.showValidation();
+                    document.getElementById("afterpay-container").scrollIntoView();
```

Comment:
This seems unrelated to the rest, is it needed?

Reply:
I added this in the places where we are calling `showValidation()` to scroll into the area where error message is shown, makes it easier for user.